src/labels/CustomLabels.labels

```diff
@@ -7327,6 +7327,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemGlobalSearchDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item Global Search in GS for End Users</shortDescription>
+        <value>When you&apos;re just getting started with NPSP, finding records can be tricky. Use Global search to search across your entire Salesforce org and find what you need fast.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemGlobalSearchLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link Label of checklist item Global Search in GS for End Users</shortDescription>
+        <value>Learn more about Search Results in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemGlobalSearchPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link Label of checklist item Global Search in GS for End Users</shortDescription>
+        <value>Find Records</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemGlobalSearchTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Global Search in GS for End Users</shortDescription>
+        <value>Find Records with Global Search</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStayTrackDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item Stay Track in GS for End Users</shortDescription>
+        <value>The activities panel helps you and your colleagues capture all work related to a particular Contact, Organization, or Opportunity. Log calls, assign tasks, send an email, and create calendar events right from the record you&apos;re working on.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStayTrackLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link Label of checklist item Stay Track in GS for End Users</shortDescription>
+        <value>Learn more about Activities in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStayTrackPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link Label of checklist item Stay Track in GS for End Users</shortDescription>
```

Comment:
I think this should be: Button label of Stay on Track checklist item in Get Started for End Users.